adafruit_pypixelbuf.py

```diff
@@ -101,16 +99,7 @@ def __init__(  # pylint: disable=too-many-locals,too-many-arguments
                 0,
             )
 
-        self._brightness = min(1.0, max(0, brightness))
```

Comment:
Why remove this cap?

I think you want to do:

```
self._brightness = 1.0
self.brightness = brightness
```

The first private assign is to make pylint happy because it enforces private attributes are set in the `__init__`.

Reply:
My mistake, I think I made it self.brightness, but then when checking it over changed it back accidentally. But thanks for the suggestion on how to satisfy pylint!